robot/Cumulus/resources/NPSP.py

```diff
@@ -52,6 +53,7 @@ def __init__(self, debug=False):
         # Turn off info logging of all http requests 
         logging.getLogger('requests.packages.urllib3.connectionpool').setLevel(logging.WARN)
         self._init_locators()
+        locator_manager.register_locators("npsp",npsp_lex_locators)
```

Comment:
What is the purpose of locator_manager ?

Reply:
using locator manager we can use our custom locators with standard library keywords for example with Selenium library keywords like Page Should Contain Element we can pass our custom locator without having the need to form the locator first. Here is a blog post about it. https://salesforce.quip.com/oyjJAcFaWtm8